src/scenes/ObservationsRouter/org/OrgObservationsRenderer.tsx

```diff
@@ -21,11 +21,13 @@ const OrgObservationsRenderer =
     goToRescheduleObservation: (observationId: number) => void,
     exportObservationCsv: (observationId: number) => void,
     exportObservationGpx: (observationId: number) => void,
+    exportObservationResults: (observationId: number) => void,
     openEndObservationModal: (observationId: Observation) => void
   ) =>
   // eslint-disable-next-line react/display-name
   (props: RendererProps<TableRowType>): JSX.Element => {
     const { column, row, value } = props;
+    const observationId = Number(row.observationId);
```

Comment:
what happens if `Number(row.observationId)` returns NaN? Is that a concern here?

Reply:
I don't think that should be possible in practice, but I'll add some code to guard against it.